roles/ceph-common/tasks/main.yml

```diff
@@ -93,10 +88,24 @@
   # Hard code this so we will skip the entire file instead of individual tasks (Default isn't Consistent)
   static: False
 
-- include: facts.yml
-- include: ./checks/check_socket.yml
+- name: get ceph version
```

Comment:
If you have a VM/docker container that is just running the rbd-mirror role, nothing is installed at this point (at least on CentOS). There are some hooks to install ceph-base on CentOS if the client role is assigned to the host so I'd imagine that should be expanded to cover any host where Ceph ansible is running.